pkg/ipamd/introspect.go

```diff
@@ -80,7 +78,7 @@ func (c *IPAMContext) ServeIntrospection() {
 			}
 
 			once.Do(func() {
-				log.Error("Error running http API: ", err)
+				log.Errorf("Error running http API: %v", err)
```

Comment:
Recommend removing this. It's not really useful to log an error when we're just going to retry the operation here. Better to handle the error in `main()`